### DISCUSSION TITLE
为什么在我用cmd启动exe的方式挂载（用5244端口）获取alist管理员密码时提示请求过多？

### DISCUSSION THREAD

**Discussion by Wn951** [AlistGo/alist#7633] [Q&A]
哈哈，没错，如标题所说（先插个视频）

https://github.com/user-attachments/assets/fc30c841-2e07-4e20-af9b-17ecf145b273

去bing翻译了一下，请求过多，不知道为什么，请各位大佬指教😁😁

**Comment by Wn951** [comment]
在线等，急！！！！！！！！！！！！

**Comment by pongfcnkl** [comment]
一 IP 在 60 分钟内请求 10 次，会出现 Too Many Requests 🚫避免滥用，请勿滥用🚫

    查看文件看视频下载不算，上述的分钟和次数是动态的
    新增优化：只会在请求发现令牌过期时去刷新令牌，更新 重启 编辑如果令牌仍然有效则不会去刷新。

**Comment by Wn951** [comment]
我想问一下，怎么重新设置啊，自从上次raidrive提示没权限读写，就想重搞，结果一直提示请求过多，我也没动什么啊
________________________________
发件人: pongfcnkl ***@***.***>
发送时间: 2024年12月9日 15:28
收件人: AlistGo/alist ***@***.***>
抄送: Wn951 ***@***.***>; Author ***@***.***>
主题: Re: [AlistGo/alist] 为什么在我用cmd启动exe的方式挂载（用5244端口）获取alist管理员密码时提示请求过多？ (Discussion #7633)


一 IP 在 60 分钟内请求 10 次，会出现 Too Many Requests 🚫避免滥用，请勿滥用🚫

查看文件看视频下载不算，上述的分钟和次数是动态的
新增优化：只会在请求发现令牌过期时去刷新令牌，更新 重启 编辑如果令牌仍然有效则不会去刷新。

**Comment by Wn951** [comment]
才发现，json文件还在😂，现在登进去又提示failed get objs: failed to list objs: driver not init
这又是怎么了？😂😂（在线等，急）
________________________________
发件人: Wn 95 ***@***.***>
发送时间: 2024年12月10日 5:29
收件人: AlistGo/alist ***@***.***>; AlistGo/alist ***@***.***>
抄送: Author ***@***.***>
主题: 回复: [AlistGo/alist] 为什么在我用cmd启动exe的方式挂载（用5244端口）获取alist管理员密码时提示请求过多？ (Discussion #7633)

我想问一下，怎么重新设置啊，自从上次raidrive提示没权限读写，就想重搞，结果一直提示请求过多，我也没动什么啊
________________________________
发件人: pongfcnkl ***@***.***>
发送时间: 2024年12月9日 15:28
收件人: AlistGo/alist ***@***.***>
抄送: Wn951 ***@***.***>; Author ***@***.***>
主题: Re: [AlistGo/alist] 为什么在我用cmd启动exe的方式挂载（用5244端口）获取alist管理员密码时提示请求过多？ (Discussion #7633)


一 IP 在 60 分钟内请求 10 次，会出现 Too Many Requests 🚫避免滥用，请勿滥用🚫

查看文件看视频下载不算，上述的分钟和次数是动态的
新增优化：只会在请求发现令牌过期时去刷新令牌，更新 重启 编辑如果令牌仍然有效则不会去刷新。

**Reply by pongfcnkl** [reply]
资源库的问题或者根文件夹id的问题

**Comment by Wn951** [comment]
那怎么解决啊😳😳
________________________________
发件人: pongfcnkl ***@***.***>
发送时间: 2024年12月10日 8:26
收件人: AlistGo/alist ***@***.***>
抄送: Wn951 ***@***.***>; Author ***@***.***>
主题: Re: [AlistGo/alist] 为什么在我用cmd启动exe的方式挂载（用5244端口）获取alist管理员密码时提示请求过多？ (Discussion #7633)


资源库的问题或者根文件夹id的问题

**Reply by pongfcnkl** [reply]
修改或者去掉

**Comment by Wn951** [comment]
huh（不太懂）
________________________________
发件人: pongfcnkl ***@***.***>
发送时间: 2024年12月10日 10:53
收件人: AlistGo/alist ***@***.***>
抄送: Wn951 ***@***.***>; Author ***@***.***>
主题: Re: [AlistGo/alist] 为什么在我用cmd启动exe的方式挂载（用5244端口）获取alist管理员密码时提示请求过多？ (Discussion #7633)


修改或者去掉

―
